readme.md

```diff
@@ -271,10 +271,16 @@ function your_custom_tracking_url_function($fields, $wc_product) {
 ### Variation Images
 By default, we don’t support any custom plugin for adding images to WC Product Variation, with this filter you can set the desired images to each variation, you can use the following snippet
 ```php
-add_filter( 'shopping_feed_variation_images', 'your_custom_variation_images_function', 10, 2 );
-
-/** @return array */
-function your_custom_tracking_url_function($images, $wc_product) {
+add_filter( 'shopping_feed_variation_images', 'your_custom_variation_images_function', 10, 3 );
+
+/** 
+ * @param array $images
+ * @param WC_Product $wc_product
+ * @param int $variation_id
+ * 
+ * @return array 
+ */
+function your_custom_tracking_url_function( $images, $wc_product, $variation_id ) {
```

Comment:
Le nom de la fonction ne correspond pas à celle présente dans le `add_filter`. Idem pour le `readme.txt`.

Reply:
Fix dans le dernier commit : https://github.com/BeAPI/shoppingfeed-for-woocommerce/pull/68/commits/7fcdebc96882a58ce1eafd089a5a7700d7899180